File: test/fixtures/node/j_patch_array.js
import fetch from 'node-fetch';

fetch('http://localhost:28139/patch', {
  method: 'PATCH',
  headers: {
    'Content-Type': 'application/x-www-form-urlencoded'
  },
  body: 'item[]=1&item[]=2&item[]=3'
});

File: test/fixtures/r/j_patch_array.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = "item[]=1&item[]=2&item[]=3"

res <- httr::PATCH(url = "http://localhost:28139/patch", httr::add_headers(.headers=headers), body = data)
